cylc/flow/config.py

```diff
@@ -1705,16 +1705,21 @@ def generate_triggers(self, lexpression, left_nodes, right, seq,
             self.taskdefs[right].add_dependency(dependency, seq)
 
         validator = XtriggerNameValidator.validate
-        for label in self.cfg['scheduling']['xtriggers']:
+        xtrigs = self.cfg['scheduling']['xtriggers']
+        for label in xtrigs:
             valid, msg = validator(label)
             if not valid:
                 raise WorkflowConfigError(
                     f'Invalid xtrigger name "{label}" - {msg}'
                 )
 
+        if self.xtrigger_mgr is not None:
+            self.xtrigger_mgr.sequential_xtriggers_default = (
+                self.cfg['scheduling']['spawn from xtriggers sequentially']
```

Comment:
(If you agree to my previous suggestion)

```suggestion
                self.cfg['scheduling']['sequential xtriggers']
```